models/RwdImage.php

```diff
@@ -218,7 +251,7 @@ public function img( $size, $attributes = array() ) {
 	 * @return string Generated html comments warnings.
 	 */
 	public function background( $selector, $size ) {
-		if ( ! $this->attachment ) {
+		if ( ! $this->attachment && $this->is_svg_image() ) {
```

Comment:
you should just allow to print this image in configured image sizes, you should not terminate method execution